net-irc/hexchat/hexchat-2.12.1.ebuild

```diff
@@ -0,0 +1,166 @@
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux"
+IUSE="dbus debug +gtk libcanberra libnotify libproxy libressl nls perl +plugins plugin-checksum plugin-doat plugin-fishlim plugin-sysinfo python spell ssl theme-manager"
+REQUIRED_USE="plugins? ( python? ( ${PYTHON_REQUIRED_USE} ) )"
+
+DEPEND="dev-libs/glib:2
+	dbus? ( >=dev-libs/dbus-glib-0.98 )
+	gtk? ( x11-libs/gtk+:2 )
+	libcanberra? ( media-libs/libcanberra )
+	libproxy? ( net-libs/libproxy )
+	libnotify? ( x11-libs/libnotify )
+	nls? ( virtual/libintl )
+	plugins? (
+		perl? ( >=dev-lang/perl-5.8.0 )
+		plugin-sysinfo? ( sys-apps/pciutils )
+		python? ( ${PYTHON_DEPS} )
+	)
+	spell? ( app-text/iso-codes )
+	ssl? (
+		!libressl? ( dev-libs/openssl:0 )
+		libressl? ( dev-libs/libressl )
+	)
+	theme-manager? ( dev-lang/mono )"
+RDEPEND="${DEPEND}
+	spell? ( app-text/enchant )"
+DEPEND="${DEPEND}
+	app-arch/xz-utils
+	virtual/pkgconfig
+	nls? ( dev-util/intltool )
+	theme-manager? ( dev-util/monodevelop )"
+
+pkg_setup() {
+	use plugins && use python && python-single-r1_pkg_setup
+	if use theme-manager ; then
+		mono-env_pkg_setup
+		export XDG_CACHE_HOME="${T}/.cache"
+	fi
+
+	if use !plugins ; then
+		local myplugins
+
+		use perl && myplugins+="perl\n"
+		use python && myplugins+="python\n"
+		use plugin-checksum && myplugins+="plugin-checksum\n"
+		use plugin-doat && myplugins+="plugin-doat\n"
+		use plugin-fishlim && myplugins+="plugin-fishlim\n"
+		use plugin-sysinfo && myplugins+="plugin-sysinfo\n"
+
+		if [[ ${myplugins} ]] ; then
+			ewarn "The following plugins/interfaces have been disabled, because"
+			ewarn "\"plugins\" USE flag is disabled. Check metadata.xml"
+			ewarn "to get more information or run \"equery u hexchat\"."
+			ewarn "\n${myplugins}"
+		fi
+	fi
+}
+
+src_configure() {
+	econf \
+		$(use_enable nls) \
+		$(use_enable ssl openssl) \
+		$(use_enable gtk gtkfe) \
+		$(use_enable !gtk textfe) \
+		$(usex plugins \
+			"$(usex python "--enable-python=${EPYTHON}" "--disable-python")" \
+			"--disable-python" \
+			) \
+		$(usex plugins \
+			"$(use_enable perl)" \
+			"--disable-perl" \
+			) \
+		$(use_enable plugins plugin) \
+		$(usex plugins \
+			"$(use_enable plugin-checksum checksum)" \
+			"--disable-checksum" \
+			) \
+		$(usex plugins \
+			"$(use_enable plugin-doat doat)" \
+			"--disable-doat" \
+			) \
+		$(usex plugins \
+			"$(use_enable plugin-fishlim fishlim)" \
+			"--disable-fishlim" \
+			) \
+		$(usex plugins \
+			"$(use_enable plugin-sysinfo sysinfo)" \
+			"--disable-sysinfo" \
+			) \
+		$(use_enable dbus) \
+		$(use_enable libnotify) \
+		$(use_enable libcanberra) \
+		$(use_enable libproxy) \
+		$(use_enable spell isocodes) \
+		$(use_enable debug) \
+		$(use_with theme-manager)
+}
+
+src_install() {
+	emake DESTDIR="${D}" \
+		UPDATE_ICON_CACHE=true \
+		UPDATE_MIME_DATABASE=true \
+		UPDATE_DESKTOP_DATABASE=true \
+		install
+	dodoc readme.md
+	prune_libtool_files --all
```

Comment:
get rid of `prune_libtool_files`, we don't want to rely on `eutils` anymore: https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives